sig/rbs_rails/app/models/announcement.rbs

```diff
@@ -254,23 +252,9 @@ class Announcement < ::ApplicationRecord
 
     def clear_title_change: () -> void
   end
-  include GeneratedAttributeMethods
 
-  def rich_text_content: () -> ActionText::RichText?
-  def rich_text_content=: (ActionText::RichText?) -> ActionText::RichText?
-  def build_rich_text_content: (untyped) -> ActionText::RichText
-  def create_rich_text_content: (untyped) -> ActionText::RichText
-  def create_rich_text_content!: (untyped) -> ActionText::RichText
-  def reload_rich_text_content: () -> ActionText::RichText?
-  def event: () -> Event
-  def event=: (Event?) -> Event?
-  def reload_event: () -> Event?
-  def build_event: (untyped) -> Event
-  def create_event: (untyped) -> Event
-  def create_event!: (untyped) -> Event
```

Comment:
[Question] Why are these lines move?

Reply:
I'm not sure. I guess this is a side effect of `rbs subtract` command.

Reply:
I see. I understood that the change is due to a generator.